index.js

```diff
@@ -162,9 +163,10 @@ module.exports = function(version, _options) {
                 instruction = options.hooks.tokenizedInstruction(instruction);
             }
 
+            var waypoint = legIndex >= 0 && legIndex !== legCount - 1 ? this.ordinalize(language, legIndex + 1) : '';
```

Comment:
This repository must have a source string before translators can provide translations in Transifex. Therefore, assuming that it’s a good idea to say “final destination”, this PR needs to add the “final destination” string to en.json. To avoid breaking the tests, I suppose we’d wait to make use of that string until enough translations are in.